frontend/src/components/page-lifeGroups/fiveEs.js

```diff
@@ -0,0 +1,156 @@
+import { Circle } from "../svgs/index";
+import { PrimaryButtonLink } from "../Button";
+
+const fiveEInfo = [
+  {
+    word: "Evangelism",
+    text1Bold: "We value ",
+    text1:
+      "God's desire for all people to have a personal and life-transforming relationship with Jesus Christ.",
+    text2:
+      ", we believe that every Christian is called to invest in the process of sharing the Gospel to those in their sphere of influence, their community, and to the nations.",
+    text2Bold: "Therefore",
+    color: "#C07560",
+  },
+  {
+    word: "Exaltation",
+    text1Bold: "We value ",
+    text1: "passionately worshiping God in and above all things.",
+    text2:
+      ", we believe in wholeheartedly giving our best in every aspect of our lives, both public and private, in order to exemplify the worthiness of God.",
+    text2Bold: "Therefore",
+    color: "#B29762",
+  },
+  {
+    word: "Edification",
+    text1Bold: "We value ",
+    text1:
+      "biblical community that encourages a deeper relationship with God and with one another.",
+    text2:
+      ", we believe in creating an environment that fosters authenticity, accountability, and opportunities to minister to one another.",
+    text2Bold: "Therefore",
+    color: "#B1A970",
+  },
+  {
+    word: "Education",
+    text1Bold: "We value ",
+    text1:
+      "cultivating disciples who know God intimately and reflect Christ’s character.",
+    text2:
+      ", we believe that God’s Word and its practical life application are foundational for our spiritual growth and journey with God.",
+    text2Bold: "Therefore",
+    color: "#65759F",
+  },
+  {
+    word: "Extension",
+    text1Bold: "We value ",
+    text1: "building up a healthy, dynamic, and vibrant church.",
+    text2:
+      ", we believe in equipping and training people for ministry so that they can impact the campus and city and influence the nations in such a way that the world cannot ignore.",
+    text2Bold: "Therefore",
+    color: "#6D8165",
+  },
+];
+
+const FiveEs = () => (
+  <div>
+    <h1 className="text-center text-[24px] font-semibold">
+      HMCC has defined the five values of biblical community found in Acts
+      2:42-47 as follows:
+    </h1>
+    <div className="flex gap-x-[44px] mx-[135px] pb-[100px]">
+      {fiveEInfo.map((item, i, index) => (
+        <span
+          key={`fiveEInfo-${index}`}
+          className="flex flex-col text-center items-center"
+        >
+          <Circle color={item.color} />
+          <div className="pt-[20px] pb-[12px] text-[24px] font-semibold">
+            {item.word}
+          </div>
+          <div className="flex flex-col gap-y-7">
+            <div className="text-[14px]">
+              <span className="font-bold">{item.text1Bold}</span>
+              {item.text1}
+            </div>
+            <div className="text-[14px]">
+              <span className="font-bold">{item.text2Bold}</span>
+              {item.text2}
+            </div>
```

Comment:
This could be done better with css modules. 
```suggestion
           {item.text}
```